### DISCUSSION TITLE
How to Change Navigation Titles

### DISCUSSION THREAD

**Discussion by ndlang** [Budibase/budibase#5328] [Help] [answered]
How would I change the Titles that show up iin Navigation:
![image](https://user-images.githubusercontent.com/17033357/162545694-fd5bf57a-7be9-4318-8bf3-fb297fb85311.png)

**Comment by cfepdan** [comment, marked as answer]
Does this help?

Navigate to Design -> Layouts
Select "Navigation Layout"

In the right-hand pane, select "Configure links"
![image](https://user-images.githubusercontent.com/28074021/162546482-f828d6c0-0a11-47dd-ae57-628ace10afc0.png)

**Reply by ndlang** [reply]
Yes, that helped. Thank you for the assist.
